### PR TITLE
Update Bonsai size and behavior

### LINKED ISSUE #996: update bonsai info
Update Bonsai storage requirements from 800gb to 1100gb.

Also maybe add note to `--bonsai-maximum-back-layers-to-load` that there's no "safe minimum" value because it doesn't impact the size of what's stored.

### PR DESCRIPTION
**Pull request by rolandtyler**
Signed-off-by: Roland Tyler <roland.tyler@consensys.net>

## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [x] you fixed all the issues raised by the tests, if any.
- [x] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change
Update Bonsai storage size.
Added note about `--bonsai-maximum-back-layers-to-load`

## Issue fixed
Fixes #996 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

### CODE REVIEW

File: docs/Concepts/Data-Storage-Formats.md [hunk 1]
```diff
@@ -49,6 +49,11 @@ To prevent this, you can limit how far Bonsai looks back while reconstructing da
 The default limit Bonsai looks back is 512. To change the parameter, use the
 [`--bonsai-maximum-back-layers-to-load`](../Reference/CLI/CLI-Syntax.md#bonsai-maximum-back-layers-to-load) option.
 
+!!! note
+
+    Using `--bonsai-maximum-back-layers-to-load` does not affect the size of the date being stored, only how far back to load.
```
Comment (matkt) [inline docs/Concepts/Data-Storage-Formats.md:54]: I think you want to Say database ?

**Review by matkt — CHANGES_REQUESTED** [review]

File: docs/Concepts/Data-Storage-Formats.md (hunk 1 above)
Comment (rolandtyler) [reply docs/Concepts/Data-Storage-Formats.md]: Fixed

**Review by matkt — APPROVED** [review]

File: docs/Concepts/Data-Storage-Formats.md [hunk 2]
```diff
@@ -49,6 +49,11 @@ To prevent this, you can limit how far Bonsai looks back while reconstructing da
 The default limit Bonsai looks back is 512. To change the parameter, use the
 [`--bonsai-maximum-back-layers-to-load`](../Reference/CLI/CLI-Syntax.md#bonsai-maximum-back-layers-to-load) option.
 
+!!! note
+
+    Using `--bonsai-maximum-back-layers-to-load` does not affect the size of the database being stored, only how far back to load.
+    This means there is no "safe minimum" value to use with this option.
```
Comment (alexandratran) [inline docs/Concepts/Data-Storage-Formats.md:55]:
```suggestion
    Using `--bonsai-maximum-back-layers-to-load` doesn't affect the size of the database being stored, only how far back to load.
    This means there's no "safe minimum" value to use with this option.
```

**Review by alexandratran — APPROVED** [review]

File: docs/Concepts/Data-Storage-Formats.md (hunk 2 above)
Comment (rolandtyler) [reply docs/Concepts/Data-Storage-Formats.md]: Fixed

**Pull request merged**